File: lista2-thehuxley/meuprimeirodicionario.py
def remover_especiais(palavra):
    especiais = ['.', '"', '(', '*', '$', '#', ':']
    for caracter in palavra:
        if(caracter not in especiais):
            palavra += caracter
        else:
            return palavra

while True:
    texto = input()
    if(texto == '-1'):
        break
    texto = texto.lower()

File: Lua/meuprimeirodicionario.lua
dicionario = {}
texto = ""

while true do
    entrada = io.read()

    if(entrada ~= "-1") then 
        texto = texto .. " " .. entrada
    else
        texto = string.lower(texto)

        for p in string.gmatch(texto, "%a+") do -- ola12-mundo0de[
            table.insert(dicionario, p)
        end

        table.sort(dicionario)

        palavra = ""
        qtd = 0

        for i, value in ipairs(dicionario) do -- am am beer clash 
            if(value == palavra) then
                qtd = qtd + 1
            else
                if(palavra ~= "") then
                    print(palavra, qtd)
                end
                palavra = value
                qtd = 1
            end   
        end  
        print(palavra, qtd) 
        break
    end 
end